integrations/_infra/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.11-slim-bookworm
+FROM python:3.12-alpine
```

Comment:
This is a rather big change, moving from debian to alpine, worth building and running the integrations locally on docker.